README.md

```diff
@@ -1,53 +1,41 @@
 # Project Name
 
-(short, 1-3 sentenced, description of the project)
+ACS Calling Sample for WPF.
 
 ## Features
 
 This project framework provides the following features:
 
-* Feature 1
-* Feature 2
-* ...
+* Embeds the ACS Calling Tutorial web application inside of a desktop application.
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
+- [npm](https://www.npmjs.com/get-npm)
+- [Node.js](https://nodejs.org/en/download/)
+- [Microsoft Edge (Chromium) Canary Chanel](https://www.microsoftedgeinsider.com/en-us/download)
+- [Visual Studio 2017 or later](https://visualstudio.microsoft.com/)
+- .NET Framework. It can be installed from Visual Studio Installer.
 
 ### Installation
 
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
-
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
-
+1. You will first need to set up and run the ACS Calling Tutorial project on localhost:5000. To do this, please follow the README file on the repo https://github.com/Azure-Samples/communication-services-web-calling-tutorial 
+2. git clone https://github.com/Azure-Samples/communication-services-web-calling-wpf-sample
+3. cd project
+4. Open WpfApp.sln in Visual Studio
```

Comment:
nit, I assume one has to "Open WpfApp.sln from `project` directory of a cloned repository"